source/app/components/EntitiesCheckbox/EntitiesCheckbox.js

```diff
@@ -60,7 +61,7 @@ export default function EntitiesCheckbox({
   let is_comprehend_medical = false
   
 
-  if (comprehendService == COMPREHEND_MEDICAL_SERVICE)is_comprehend_medical=true
+  if (comprehendService == COMPREHEND_MEDICAL_SERVICE && ENABLE_COMPREHEND_MEDICAL)is_comprehend_medical=true
```

Comment:
This is not needed as we are already conditionally rendering this component in `view.js` based on the feature flag

Reply:
Thanks for pointing it out